MTA-6893 Document Hub OIDC authentication (#382)
migtools/mta-documentation: docs/topics/mta-install/con_roles-personas-users-permissions.adoc

A user can have more than one role. Each role must correspond to a specific persona:

> * The `admin` role corresponds to the Administrator persona. The Administrator has all permissions that Architects and Migrators have. Administrators can also create application-wide configuration parameters that other users can consume but cannot change or view, for example, Git credentials or Maven `settings.xml` files.

Reviewer: ```suggestion
* The `admin` role corresponds to the administrator persona. Administrators have all the permissions of architects and migrators. They can also create application-wide configuration parameters that other users can use but cannot change or view, such as Git credentials or Maven `settings.xml` files.
```

> * The `architect` role corresponds to the Architect persona. An Architect is a technical lead for the migration project. Architects can run assessments and can create and modify applications and information related to the applications. Architects cannot modify or delete sensitive information, but can consume such information. For example, architects can associate an existing set of credentials to the repository of a specific application.

Reviewer: ```suggestion
* The `architect` role corresponds to the architect persona. An architect is a technical lead for the migration project. They can run assessments, and create and modify applications and related information. They cannot modify or delete sensitive information, but can view it. For example, they can associate an existing set of credentials with a specific application repository.
```

> * The `migrator` role corresponds to the migrator persona. A migrator can analyze applications but cannot create, modify, or delete the them. 

Reviewer 2: cannot create, modify, or delete them

Author: Modified the sentence here, thanks!

> * The `migrator` role corresponds to the Migrator persona. A Migrator can analyze applications. However, the Migrator cannot create, modify, or delete the applications. 

Reviewer: ```suggestion
* The `migrator` role corresponds to the migrator persona. A migrator can analyze applications, but cannot create, modify, or delete them.
```

> * The `project-manager` role corresponds to the Project Manager persona. Project managers can create, read, update, and delete migration waves. They can also create {mta-dl-full} resources and adoption plans, and update business stakeholders included in a project. Project managers have read-only access to other {ProductShortName} resources. 

Reviewer: ```suggestion
* The `project-manager` role corresponds to the project manager persona. Project managers can create, read, update, and delete migration waves. They can also create {mta-dl-full} resources and adoption plans, and update business stakeholders included in a project. They have read-only access to other {ProductShortName} resources.
```

> {ProductShortName} has two views, *Administration*, and *Migration*. Only an Administrator can access the *Administration* view. Architects and Migrators cannot see or access the *Administration* view. Administrators can perform all actions supported by the *Migration* view. Architects and Migrators can see all elements of the *Migration* view. However, whether Architects and Migrators can perform actions in the *Migration* view depends on the permissions granted to their role.

Reviewer: ```suggestion
{ProductShortName} has two views: *Administration* and *Migration*. Only an administrator can access the *Administration* view. Administrators can perform all actions in the *Migration* view. Architects and migrators can see all elements in this view, but their ability to perform actions depends on their role permissions.
```
